examples/lush_tutorial.lua

```diff
@@ -215,12 +204,10 @@ local theme = lush(function()
   }
 end)
 
--- return our parsed theme for extension or use else where.
+-- return our parsed theme as a plugin or use elsewhere.
```

Comment:
extension is the intent here, as in use via `lush.extend`.

Reply:
Probably it should be

-- Return our parsed theme for use and that's the basics of using Lush!
-- The parsed theme can be applied as a neovim theme, or extended further via lush,
-- or used elsewhere such as in other lua runtimes.

Reply:
Yeah